bin/import.ml

```diff
@@ -54,9 +53,9 @@ module Main = struct
 
   let setup ~log ?external_lib_deps_mode (common : Common.t) =
     scan_workspace ~log common
-    >>= init_build_system
-          ?external_lib_deps_mode
-          ?only_packages:common.only_packages
+    |> Fiber.bind ~f:(init_build_system
```

Comment:
Why this change? I find the original code simpler

Reply:
I'll revert it. It's because I don't see `>>=` and `>>|` to be very useful anymore.

Reply:
And also because we now don't need an `open`.

Reply:
If you want to remove the `>>=`, I would suggest to just use `let*` for consistency. Introducing intermediate variable often helps readability anyway.